src/styles/cards/cards_blocks/card.scss

```diff
@@ -0,0 +1,91 @@
+@use "./cards_utils/__variables" as *;
+
+.card {
+  display: flex;
+  flex-direction: column;
+  width: 200px;
+  background-color: #fff;
```

Comment:
Organize your styles
I recommend you take a look at these styles code and use it in the feature 
[9elements](https://9elements.com/css-rule-order) or  [htmlAcademy](https://codeguide.academy/html-css.html#css-order)